code/game/machinery/doors/firedoor.dm

```diff
@@ -250,10 +250,10 @@ var/global/list/alert_overlays_global = list()
 		investigation_log(I_ATMOS, "[density ? "closed" : "opened"] [alarmed ? "while alarming" : ""] by [user.real_name] ([formatPlayerPanel(user, user.ckey)]) at [formatJumpTo(get_turf(src))]")
 
 /obj/machinery/door/firedoor/CtrlClick(mob/user)
-	if(isAdminGhost(user))
-		attack_ai(user,TRUE)
-	else
-		..()
+    if(isrobot(user) || isAdminGhost(user))
+        attack_ai(user, TRUE)
+    else
+        ..()
```

Comment:
indent with tab instead of space

Reply:
I DIDN'T EVEN NOTICE THIS WHAT THE HELL? 🤨
fixed this in the latest commit. blame github user mentgah